cirq/sim/mux.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2019 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Sampling/simulation methods that delegate to appropriate simulators."""
+
+from typing import List, Type, Union, Optional
+
+import numpy as np
+
+from cirq import circuits, protocols, study, schedules
+from cirq.sim import sparse_simulator, density_matrix_simulator
+
+
+def sample(program: Union[circuits.Circuit, schedules.Schedule],
+           *,
+           param_resolver: Optional[study.ParamResolver] = None,
+           repetitions: int = 1,
+           dtype: Type[np.number] = np.complex64) -> study.TrialResult:
+    """Simulates sampling from the given circuit or schedule.
+
+    Args:
+        program: The circuit or schedule to sample from.
+        param_resolver: Parameters to run with the program.
+        repetitions: The number of samples to take.
+        dtype: The `numpy.dtype` used by the simulation. Typically one of
+            `numpy.complex64` or `numpy.complex128`.
+            Defaults to fast, i.e. to `numpy.complex64`.
```

Comment:
nit: Defaults to fast, i.e. ... -> Favors speed over precision by default, i.e. ...

Reply:
Done.